.pipelines/pipeline.yaml

```diff
@@ -128,90 +128,138 @@ stages:
+            name: "Python"
+            displayName: "Install python3.9"
+            condition: succeeded()
+          - bash: |
+              set -e
+              python3.9 -m pip install requests
+              python3.9 -m pip install pytz
+              python3.9 -m pip install azure-storage-blob
+              python3.9 -m pip install azure-storage-queue
+              python3.9 -m pip install azure-kusto-ingest
+              python3.9 -m pip install azure-kusto-data
+              python3.9 -m pip install pandas
+            name: "Modules"
+            displayName: "Install modules"
+            condition: succeeded()
+
+          - task: BuildQualityChecks@8
+            inputs:
+              checkCoverage: true
+              coverageFailOption: "build"
+              coverageType: "lines"
+              allowCoverageVariance: true
+              coverageVariance: 0.1
+              fallbackOnPRTargetBranch: false
+              baseBranchRef: "master"
+            displayName: "Coverage Policy Check"
+
+          - script: |
+              env | sort
+              python3.9 publish_build_metric.py
+            displayName: "Publish build metrics to console"
+            workingDirectory: "$(Build.SourcesDirectory)/.pipelines/scripts/"
+            env:
+              KUSTO_INGEST_CLUSTER_URL: $(KUSTO_INGEST_CLUSTER_URL)
+              KUSTO_DB_NAME: $(KUSTO_DB_NAME)
+              KUSTO_TABLE_NAME: $(KUSTO_TABLE_NAME)
+              KUSTO_SERVICE_PRINCIPAL_ID: $(KUSTO_SERVICE_PRINCIPAL_ID)
+              KUSTO_SERVICE_PRINCIPAL_PASSWORD: $(KUSTO_SERVICE_PRINCIPAL_PASSWORD)
+              TENANT_ID: $(TENANT_ID)
+            condition: always()
+
   - template: windows-image.yaml
 
   - stage: containerize
     displayName: Build Images
     dependsOn:
-    - setup
-    - build_and_test
+      - setup
+      - build_and_test
     jobs:
-    - job: containerize 
-      displayName: Build Images
-      variables:
-        TAG: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.Tag'] ]
-      pool:
-        name: "$(BUILD_POOL_NAME_DEFAULT)"
-      strategy:
-        matrix:
-          cni_manager_linux_amd64:
-            arch: amd64
-            os: linux
-            name: cni-manager
-          cni_manager_linux_arm64:
-            arch: arm64
-            os: linux
-            name: cni-manager
-          cns_linux_amd64:
-            arch: amd64
-            os: linux
-            name: cns
-          cns_linux_arm64:
-            arch: arm64
-            os: linux
-            name: cns
-          npm_linux_amd64:
-            arch: amd64
-            os: linux
-            name: npm 
-          npm_linux_arm64:
-            arch: arm64
-            os: linux
-            name: npm
-      steps:
-      - template: containers/container-template.yaml
-        parameters:
-          name: $(name)
-          os: $(os)
-          arch: $(arch)
-          tag: $(TAG)
+      - job: containerize
```

Comment:
is this whitespace change from some autoformatter? i think lists should be dedented to the same level as their parent item for max readability

Reply:
Yeah it was, I can change it